src/components/Transactions.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+export default function Transactions() {
+    return (
+        <section className="bg-[#FFF7ED] px-5 ">
+            <div className="max-w-6xl mx-auto">
+                <div className=''>
+                    <div className="flex justify-between max-w-6xl mx-auto  items-center py-4 ">
+                        <p className="text-2xl font-bold">Your Transaction</p>
+                        <p className="text-xl font-semibold">View All</p>
+                    </div>
+                </div>
+                {[1, 2, 3].map((val, index) => {
```

Comment:
Could you use a more descriptive name for the `val` argument? You can remove the `index` argument as you are not using it. Lastly, as you are mapping, add a key to the elements.